Exercise_B/assets/style/stylesheet.css

```diff
@@ -1 +1,156 @@
 /* Your CSS code here */
+nav {
+    height: 80px;
+    background: white;
+    padding: 0px 20px;
+    display: flex;
+    align-items: center;    
+}
+
+nav img{
+    height: 70px;
+}
+
+header {
+    background-image:    linear-gradient(
+        rgba(0, 0, 0, 0.6), 
+        rgba(0, 0, 0, 0.6)
+      ), url("/Exercise_B/assets/images/flowershop.jpg");
+    height: 800px;
+    background-position: center; /* Center the image*/
+    background-repeat: no-repeat; /* Do not repeat the image*/
+    background-size: cover; /* Resize the background image to cover the entire container */
+    align-content: center;
+    justify-content: center;
+    display: flex;
+    color: white;
+    font-weight: bold;
+    flex-flow: row wrap;
+}
+
+.break {
+    flex-basis: 100%;
+    height: 20px;
+}
+
+.button {
+    background-color: greenyellow;
+    height:60px;
+    width: fit-content;
+    text-align: center;
+    padding: 12px 15px 10px 15px;
+    border-radius: 30px;
+}
+
+.button a{
+    text-decoration: none;
+    color: white;
+}
+
+section {
+    background-color: aquamarine;
+    padding: 100px;
+    font-size: 200%;
+    color:gray;
+}
+
+.col-sm-4, .col-sm-8, .col-lg-2, .col-lg-4{
+    padding: 50px 0px;
+}
+
+.grid {
+    align-self: center;
+    margin: auto;
+    display:grid;
+    max-width: 1200px;
+    grid-template-columns: 1fr 2fr 1fr 2fr;
+    grid-auto-rows: 250px;
+    column-gap: 40px;
+}
+
+/*Widgets*/
+
+footer{
+    background:#eee;
+    padding:40px 40px 20px 40px;
+}
+
+.rounded {
+  -webkit-border-radius: 3px !important;
+  -moz-border-radius: 3px !important;
+  border-radius: 3px !important;
+}
+
+.mini-stat {
+  padding: 15px;
+  margin-bottom: 20px;
+}
+
+.mini-stat-icon {
+  width: 60px;
+  height: 60px;
+  display: inline-block;
+  line-height: 60px;
+  text-align: center;
+  font-size: 30px;
+  background: none repeat scroll 0% 0% #EEE;
+  border-radius: 100%;
+  float: left;
+  margin-right: 10px;
+  color: #FFF;
+}

```

Comment:
Consider using responsive units so that the website works on any size of screen!